lib/app/css/styleguide-app.css

```diff
@@ -344,40 +345,255 @@ markup:
 Styleguide 3.1.1
  */
 
-
+.sg.full-height {
+  min-height: 100vh;
+}
 
 .sg.sg-wrapper {
   position: relative;
-  padding: var(--wrapper-vertical-padding) var(--content-margin) var(--wrapper-vertical-padding) var(--min-content-margin);
+  overflow: hidden;
+  max-width: 68em;
+  margin: 0 auto;
   padding: $wrapper-vertical-padding $content-margin $wrapper-vertical-padding $min-content-margin;
-  max-width: $content-max-width;
-  margin-left: auto;
-  margin-right: auto;
-
+  @media (--mobile) {
+    padding: 0;
+  }
   &.designerToolVisible {
     margin-right: var(--designer-tool-width);
     margin-right: $designer-tool-width;
   }
-}
 
-.sg.full-height {
-  min-height: 100vh;
-}
+  .sg.sg-body {
+    display: inline-block;
+    width: 70%;
+    min-height: 680px;
+    height: 100%;
+    @media (--mobile) {
+      padding: 0.25em;
+      padding-top: 30px;
+      width: 100%;
+    }
+    // Remove top margin from first header in markdown
+    .sg.sg-heading:nth-of-type(1) {
+      margin-top: 0;
+      padding-top: 0;
+    }
+  }
 
-.sg.sg-body {
-  width: 100%;
-  height: 100%;
-  @media (--mobile) {
-    padding: 0.25em;
+  .sg.full-width {
+    width: 100%;
   }
 
-  // Remove top margin from first header in markdown
-  .sg.sg-heading:nth-of-type(1) {
-    margin-top: 0;
-    padding-top: 0;
+  .sg.sg-side-nav {
+    float: left;
+    display: inline-block;
+    width: 30%;
+    opacity: 1;
+    @media (--mobile) {
+      width: 100%;
+      padding-top: 30px;
+      z-index: 99999;
+      background: #8b8b8b;
+      overflow: scroll;
+      transition: opacity 400ms cubic-bezier(0.000, 0.995, 0.990, 1.000);
```

Comment:
Two different notations for time: 400ms and .4s. Perhaps, only either is enough?